### PR TITLE
add owner id to end time filter

### PR DESCRIPTION
**Pull request by JWeston-HO**
Add owner id to the end time filter. Time periods could be seen by other users. By adding the owner id to the end time filter

### CODE REVIEW

File: src/utils/filters/time-entry-filter/timeEntryFilterBuilder.js [hunk 1]
```diff
@@ -17,10 +17,12 @@ const filterTimeEntriesOnDate = (resourceDateProperty, date) => {
 
 export const buildTimeEntriesFilter = (date, userId) => {
   const startDateFilterCondition = joinAndConditions(
+    "ownerId=='" + userId + "'",
     ...filterTimeEntriesOnDate('actualStartTime', date)
```
Comment (laurieboyes) [inline src/utils/filters/time-entry-filter/timeEntryFilterBuilder.js:21]: Not sure if the line breaks will go weird but I’m a fan of template strings over concatenation. Would this work?
```suggestion
    `ownerId=='${userId}'${...filterTimeEntriesOnDate('actualStartTime', date)}`
```

Comment (JWeston-HO) [reply src/utils/filters/time-entry-filter/timeEntryFilterBuilder.js]: Not sure the above would work. We need to add the && in-between filter arguments. I have pushed a changed that stores "ownerId=='" + userId + "'" as a const. As this is used multiple times.

File: src/utils/filters/time-entry-filter/timeEntryFilterBuilder.js [hunk 2]
```diff
@@ -16,11 +16,15 @@ const filterTimeEntriesOnDate = (resourceDateProperty, date) => {
 };
 
 export const buildTimeEntriesFilter = (date, userId) => {
+  const ownerAndUserId = "ownerId=='" + userId + "'";
```
Comment (laurieboyes) [inline src/utils/filters/time-entry-filter/timeEntryFilterBuilder.js:19]: It’s an even more pedantic comment than it was before but could _this_ be a template string?
```suggestion
  const ownerAndUserId = `ownerId==${userId}'`;
```

Comment (JWeston-HO) [reply src/utils/filters/time-entry-filter/timeEntryFilterBuilder.js]: yes it can committed.

**Review by laurieboyes — APPROVED** [review]

**Review by joannafawl — APPROVED** [review]

**Pull request merged**